Project/Book.swift

```diff
@@ -19,6 +19,9 @@ struct Book {
     var image: String
     var year: String
     var genre: String
+    static public let available = "Available"
+    static public let unavailable = "Unavailable"
```

Comment:
We will surely refactor for this once we get to the localization card in Trello 👍